packages/databyss-ui/stories/Errors/sourcesError.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import ErrorFallback from '@databyss-org/ui/components/Notify/ErrorFallback'
+import { ViewportDecorator } from '../decorators'
+
+storiesOf('Notify//Sources', module)
```

Comment:
Doesn't need its own story file. We can put this in `databyss-ui/stories/Components/notify.stories.js`